en/docs/references/ballerina-expression-syntax.md

```diff
@@ -380,7 +379,7 @@ To create a nested JSON object, use the following:
 
 JSON array values must be of any of the json types: `string`, `number`, `boolean`, `object`, `array`, or `null`. You can define a JSON array in one of the following two ways:
     - Using a [`json` type variable](#simple-values-in-JSON).  
-    - Using the variable type `other` and `json[]` as the `other type`.
+    - Using the variable type `other` and `json[]` as the `Other Type`.
```

Comment:
What is meant by the type `other`?

Reply:
In the variable drop down, there's a variable type called other.